stwcs/updatewcs/astrometry_utils.py

```diff
@@ -360,12 +410,252 @@ def getObservation(self, observationID):
 
             return headerlets, best_solution_id
 
+    def find_gsc_offset(self, obsname, refframe="ICRS"):
```

Comment:
Moved to be a stand-alone function in 1d9dc68.  It can be useful for anyone wanting to get a record of the astrometric corrections applied to their observations for the a priori WCSs.